apps/minifront/src/components/shared/asset-selector.tsx

```diff
@@ -0,0 +1,126 @@
+import {
```

Comment:
Almost all of this is copied from `AssetOutSelector`, but with different prop names. I'll highlight the changes.